test/stdlib/StringBridge.swift

```diff
@@ -39,6 +39,36 @@ func expectCocoa(_ str: String,
   }
 }
 
+class FakeString : NSString {
+  var contents: NSString
+  
+  override init() {
+    contents = NSString()
+    super.init()
+  }
+  
+  required init?(coder aDecoder: NSCoder) {
+    guard let guts = NSString(coder: aDecoder) else {
+      return nil
+    }
+    contents = guts
+    super.init()
+  }
+  
+  init(str: String) {
+    contents = str as NSString
+    super.init()
+  }
+  
+  override var length: Int {
+    return contents.length
+  }
+  
+  override func character(at index: Int) -> unichar {
+    return contents.character(at: index)
+  }
+}
```

Comment:
Hmm, we're collecting a lot of these in individual test files. I'll try to remember to open a JIRA for StdlibUnittest to provide common ones when that's back up.